resources/templates/account.html

```diff
@@ -123,19 +125,37 @@ <h2>Fund your Raiden Account with ETH</h2>
    }
  }
 
+ function updateNeededEth(balance) {
+   neededEthAmount = ETHEREUM_REQUIRED_AMOUNT - balance.ETH.as_wei;
```

Comment:
does js takes the outer scope variable automatically?

Reply:
Yes it does. Only if you would shadow the same variable name in the inner scope it would not.